app/Http/Controllers/StatusController.php

```diff
@@ -418,4 +419,32 @@ public static function getFutureCheckins(): Paginator {
                          $query->where('departure', '>=', date('Y-m-d H:i:s', strtotime("+20min")));
                      })->simplePaginate(15);
     }
+
+    public static function createStatus(
+        User $user,
+        bool $business,
+        $visibility,
```

Comment:
Is there a reason why the type of $visibility is not specified?

Reply:
Yes, because with the current enum-types this will fail. Especially b/c we can't use typecasting.

Reply:
simply use int for this? Or am I wrong?

Reply:
I'm not sure about that since the enums aren't fully int and this would be a bit stupid to exclude them. :D